### PR TITLE
People and Address entity

### PR DESCRIPTION
**Pull request by devzgabriel**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -8,27 +8,65 @@ datasource db {
...
+  name String
+  cpf  String
+
+  createdAt DateTime  @default(now())
+  updatedAt DateTime? @updatedAt
+  deletedAt DateTime?
+
+  user     User?
+  contacts Contact[]
+  address  Address[]
```
Comment (SampaioLeal) [inline prisma/schema.prisma:37]:
```suggestion
  addresses  Address[]
```

File: prisma/schema.prisma [hunk 2]
```diff
@@ -8,27 +8,65 @@ datasource db {
...
+  roles Role[]
+
+  person   Person @relation(fields: [personId], references: [id])
+  personId String @unique
+}
+
+model Person {
+  id   String @id @default(uuid())
+  name String
+  cpf  String
```
Comment (SampaioLeal) [inline prisma/schema.prisma:29]:
```suggestion
  identifier  String
```

File: prisma/schema.prisma [hunk 3]
```diff
@@ -8,27 +8,65 @@ datasource db {
...
+
+model Address {
+  id           String @id @default(uuid())
+  street       String
+  number       Int
+  neighborhood String
+  city         String
+  state        String
+  country      String
+  cep          String
```
Comment (SampaioLeal) [inline prisma/schema.prisma:62]:
```suggestion
  zipcode          String
```

**Review by SampaioLeal — CHANGES_REQUESTED** [review]

File: prisma/seed.ts [hunk 4]
```diff
@@ -6,7 +6,38 @@ import { v4 as uuid } from 'uuid';
 const prisma = new PrismaClient();
 const logger = new Logger('PrismaSeed');
 
-async function createUsers() {
+async function createPeople() {
+  const personId1 = uuid();
+  const personId2 = uuid();
+  const personId3 = uuid();
+
+  const people = [
```
Comment (JoaoptGaino) [inline prisma/seed.ts:14]: Don't forget to change 'cpf' to 'identifier' here in the seed too.

**Review by SampaioLeal — APPROVED** [review]

**Pull request merged**